src/somef/process_files.py

```diff
@@ -171,33 +171,55 @@ def process_repository_files(repo_dir, metadata_result: Result, repo_type, owner
                         if analysis:
                             workflow_url_gitlab = get_file_link(repo_type, file_path, owner, repo_name, repo_default_branch,
                                                         repo_dir, repo_relative_path, filename)
-                            metadata_result.add_result(constants.CAT_WORKFLOWS,
-                                                       {
+                            metadata_result.add_result(constants.CAT_CONTINUOS_INTEGRATION,
+                                                    {
                                                         constants.PROP_VALUE: workflow_url_gitlab,
                                                         constants.PROP_TYPE: constants.URL
-                                                    }, 1, constants.TECHNIQUE_FILE_EXPLORATION)                
-                else:
-                    if filename.endswith(".ga") or filename.endswith(".cwl") or filename.endswith(".nf") or (
-                            filename.endswith(".snake") or filename.endswith(
-                        ".smk") or "Snakefile" == filename_no_ext) or filename.endswith(".knwf") or filename.endswith(
-                        ".t2flow") or filename.endswith(".dag") or filename.endswith(".kar") or filename.endswith(
-                        ".wdl"):
-                        analysis = extract_workflows.is_file_workflow(os.path.join(repo_dir, file_path))
-                        if analysis:
+                                                    }, 1, constants.TECHNIQUE_FILE_EXPLORATION)
+                        elif extract_workflows.is_file_workflow(os.path.join(repo_dir, file_path)):
                             workflow_url = get_file_link(repo_type, file_path, owner, repo_name, repo_default_branch,
-                                                        repo_dir, repo_relative_path, filename)
+                                                            repo_dir, repo_relative_path, filename)
                             metadata_result.add_result(constants.CAT_WORKFLOWS,
-                                                    {
-                                                        constants.PROP_VALUE: workflow_url,
-                                                        constants.PROP_TYPE: constants.URL
-                                                    }, 1, constants.TECHNIQUE_FILE_EXPLORATION)
-                 
+                                                        {constants.PROP_VALUE: workflow_url, constants.PROP_TYPE: constants.URL},
+                                                        1, constants.TECHNIQUE_FILE_EXPLORATION)
+                            
+                    elif repo_type == constants.RepositoryType.GITHUB:
+                        if file_path.startswith(".github/workflows/"):
+                            category = constants.CAT_WORKFLOWS
```

Comment:
This is not correct. Everything in the /github/workflows folder is continuous integration